crates/superposition/src/main.rs

```diff
@@ -167,6 +167,7 @@ async fn main() -> Result<()> {
             .wrap(
                 actix_web::middleware::DefaultHeaders::new()
                     .add(("X-SERVER-VERSION", cac_version.to_string()))
+                    .add(("Cache-Control", "no-store".to_string()))
```

Comment:
Is this restricted to all responses or only front-end ?

Reply:
I think this will affect the backend routes as well.

Reply:
@knutties its mainly for backend , 
basically client calls backend apis to fetch data , now data gets cached in clients and served from cache which leads to inconsistent results , so we should have cache control for backend apis first